wremnants/CardTool.py

```diff
@@ -741,16 +756,19 @@ def addPOISumGroup(self, groupName, members, groupLabel="sumGroup"):
 
     def writeLnNSystematics(self):
         nondata = self.predictedProcesses()
+        nondata_chan = {chan: nondata.copy() for chan in self.channels}
```

Comment:
What is this actually doing?

Reply:
Ok, I find the code a little confusing, but the motivation is clear enough

Reply:
Yeah, I agree, this is also why time ago I was suggesting to modify CardTool to have a single loop on channels, if they exist, rather than have each member function loop on channels. In that case these channel specific settings are needed only once, rather than appearing within a few member functions